# ROS headers from request will be passed as HTTP headers
HttpRequestHeaders headers
---
# The requests headers are returned here, as part of the body of the response
HttpRequestHeaders headers
